google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -68,7 +68,6 @@
 import com.google.common.base.Function;
 import com.google.common.base.MoreObjects;
 import com.google.common.base.Preconditions;
-import com.google.common.base.Supplier;
```

Comment:
This is not directly related to this change, but we should start to replace the use of `com.google.common.base.Supplier` with `java.util.function.Supplier`.